docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,218 @@
+
+Installation
+============
+
+For the first step of our journey, let's install MCDR
+
+.. note::
+
+    We assume you have some really basic knowledge of Python and pip.
+
+    If you have not, Google is your friend
+
+Requirements
+------------
+
+.. image:: https://img.shields.io/pypi/pyversions/mcdreforged.svg
+   :alt: Python version
+
+.. image:: https://img.shields.io/pypi/v/mcdreforged.svg
+   :alt: PyPI version
+
+MCDR is written and runs in Python 3. Make sure you have Python 3.8 or later with pip3 installed
+
+The detailed Python version requirements are as shown in the table below
+
+.. list-table::
+   :header-rows: 1
+
+   * - MCDR version
+     - Python requirement
+   * - < 2.10
+     - >= 3.6
+   * - >= 2.10
+     - >= 3.8
+
+Install using pip
+-----------------
+
+MCDR is available in `PyPI <https://pypi.org/project/mcdreforged>`__, it can reasonably be installed via the ``pip`` command:
+
+.. tab:: Windows
+
+    .. code-block:: bat
+
+        pip install mcdreforged
+
+.. tab:: Linux
+
+    .. code-block:: bash
+
+        pip3 install mcdreforged
+
+Externally managed environment
+------------------------------
+
+If you're using Windows, the command above should work fine, MCDR will be installed to global environment - you may ignore this section
+
+For Linux and Mac OS, it's not recommended to install MCDR system-wide, because it can cause conflicts with other Python packages and affect system dependencies
+
+System-wide install also makes version management difficult 
+and requires administrator privileges, increasing security risks
+
+For the same reason, you may get a ``externally-managed-environment`` error on pip install
+
+See `PEP 668 <https://peps.python.org/pep-0668/>`__ for the detailed specification
+
+It's safer to keep the installation isolated. As workarounds, there're multiple options for you. For a conclusion:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Method
+      - Pros
+      - Cons
+    * - pip
+      - Native, always available
+      - Not isolated, may affect global packages, requires admin privilege
```

Comment:
pip does not always require admin privilege. You can let pip install package to user-side (e.g. `~/.local/` in linux)

e.g., in RHEL 9, with the python 3.9 provided by the OS, when running `pip` with non-root user, you will get:

```
$ pip3 install wcwidth
Defaulting to user installation because normal site-packages is not writeable
Collecting wcwidth
  Downloading wcwidth-0.2.13-py2.py3-none-any.whl (34 kB)
Installing collected packages: wcwidth
```

The package will be automatically installed to `~/.local/`, no admin privilege is required